src/Mvc/Mvc.Core/src/MvcOptions.cs

```diff
@@ -103,7 +103,8 @@ public MvcOptions()
         public FormatterCollection<IInputFormatter> InputFormatters { get; }
 
         /// <summary>
-        /// Gets or sets the flag to buffer the request body in input formatters. Default is <c>false</c>.
+        /// Gets or sets the flag that determines if buffering is disabled for input formatters that
```

Comment:
```suggestion
        /// Gets or sets a value that determines if buffering is disabled for input formatters that
```

Reply:
I can't tell exactly what the final outcome shows. The Win Zip Files won't work

Reply:
The blue highlights?